controllers/ibmpowervsmachine_controller.go

```diff
@@ -198,6 +198,23 @@ func (r *IBMPowerVSMachineReconciler) getOrCreate(scope *scope.PowerVSMachineSco
 	return instance, err
 }
 
+// handleLoadBalancerPoolMemberConfiguration handles loadbalancer pool member creation flow.
+func (r *IBMPowerVSMachineReconciler) handleLoadBalancerPoolMemberConfiguration(machineScope *scope.PowerVSMachineScope) (ctrl.Result, error) {
+	if !util.IsControlPlaneMachine(machineScope.Machine) {
+		return ctrl.Result{}, nil
```

Comment:
May be lets use log.V(3) to log that we are skipping loadbalancer configuration as its not control plane machine

Reply:
Done